tests/containers/run_container_in_k3s.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Installs a local kubernetes engine
```

Comment:
The summary suggests something different than the filename.

Reply:
done